docs/_ext/autorobusta.py

```diff
@@ -119,7 +120,8 @@ def __document_field_content(cls, field: ModelField, show_code: bool, show_optio
         if show_code:
             content.extend(cls.__document_field_example(field))
 
-        if typing.get_origin(field.type_) == typing.Union:
+        # TODO: Will not work for 3.7
```

Comment:
Will this impact users trying to build robusta on python 3.7?

Reply:
The thing is - it was in the codebase, it is not my change.
PyRight raised an error since I was using python 3.7 as the lowest supported version, so it found that this might not work. I think there is at least one more place in the code that will not work with 3.7, that I told @arikalon1 about. Maybe we should discuss on how to better solve it if we should.

And, just for reminder, I left those comments.